source/scale/scale-to-88000-users.rst

```diff
@@ -15,15 +15,20 @@ This page describes the Mattermost reference architecture designed for the load
 Requirements
 ------------
 
-+------------------------+----------+------------------+----------------+------------------+
-| **Resource Type**      | **vCPU** | **Memory (GiB)** | **# of Nodes** | **AWS Instance** |
-+========================+==========+==================+================+==================+
-| Mattermost Application | 16       | 32               | 6              | c6i.4xlarge      |
-+------------------------+----------+------------------+----------------+------------------+
-| RDS Writer             | 16       | 128              | 1              | db.r6g.4xlarge   |
-+------------------------+----------+------------------+----------------+------------------+
-| RDS Reader             | 16       | 128              | 4              | db.r6g.4xlarge   |
-+------------------------+----------+------------------+----------------+------------------+
++------------------------+-----------+----------------+-------------------+
+| **Resource Type**      | **Nodes** | **vCPU/        | **AWS Instance**  |
+|                        |           | Memory (GiB)** |                   |
++========================+===========+================+===================+
+| Mattermost Application | 6         | 16/32          | c6i.4xlarge       |
++------------------------+-----------+----------------+-------------------+
+| RDS Writer             | 1         | 16/128         | db.r6g.4xlarge    |
++------------------------+-----------+----------------+-------------------+
+| RDS Reader             | 4         | 16/128         | db.r6g.4xlarge    |
++------------------------+-----------+----------------+-------------------+
+| Elasticsearch Node     | X         | 4/32           | r6g.xlarge.search |
```

Comment:
X? @cwarnermm can I help here?

Reply:
@nab-77 - you're correct that any ``X`` indicates a TBD value I couldn't clearly identify via the [v2](https://mattermost.atlassian.net/wiki/spaces/XYZ/pages/2809004081/Ceiling+Tests+V2) results.